utilities/src/serialize/impls.rs

```diff
@@ -594,6 +651,25 @@ mod test {
         }
     }
 
+    fn test_serialize_without_len<T: PartialEq + std::fmt::Debug + CanonicalSerialize + CanonicalDeserialize>(
+        data: Vec<T>,
+    ) {
+        let combinations = [
+            (Compress::No, Validate::No),
+            (Compress::Yes, Validate::No),
+            (Compress::No, Validate::Yes),
+            (Compress::Yes, Validate::Yes),
+        ];
+        for (compress, validate) in combinations {
+            let len = serialized_vec_size_without_len(&data, compress);
+            let mut serialized = vec![0; len];
+            serialize_vec_without_len(&data, &mut &mut serialized[..], compress).unwrap();
```

Comment:
another case of undesirable `&mut &mut` followed by `&mut &` below

Reply:
c6cd41f8b